test/v1/controllers/requestParsers/validators/AmendOtherDeductionsValidatorSpec.scala

```diff
@@ -29,7 +29,7 @@ import v1.models.request.amendOtherDeductions.AmendOtherDeductionsRawData
 class AmendOtherDeductionsValidatorSpec @Inject()(implicit appConfig: AppConfig, currentTaxYear: CurrentTaxYear) extends UnitSpec {
```

Comment:
I know you didn't change this but the Inject stuff is unnecessary (also remove any unused imports please).

```suggestion
class AmendOtherDeductionsValidatorSpec extends UnitSpec {
```